drivers/console/aarch64/skeleton_console.S

```diff
@@ -60,8 +61,12 @@ func console_xxx_register
 	 * Keep console_t pointer in x0 for later.
 	 */
 
-	/* Macro to finish up registration and return (needs valid x0 + x30). */
-	finish_console_register xxx
+	/*
+	 * Macro to finish up registration and return (needs valid x0 + x30).
+	 * If any of the argument is blank, then the corresponding callback
```

Comment:
nit: Maybe clarify that you'll need to use the _driver_xxx=function form in that case, just leaving one out in the middle will not work well.